core/src/test/java/org/apache/iceberg/rest/RESTCatalogAdapter.java

```diff
@@ -89,11 +94,13 @@ public class RESTCatalogAdapter implements RESTClient {
 
   private final Catalog catalog;
   private final SupportsNamespaces asNamespaceCatalog;
+  private final ViewCatalog viewCatalog;
```

Comment:
Nit: `asViewCatalog` to match the `asNamespaceCatalog` name? That way it's more clear that this isn't a completely different view catalog.